new/LIT6290Tafassehi.xml

```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6290Tafassehi" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ሰአሊ፡ ለነ፡ ማርያም፡ ማኅቶተ፡ ዓለም፡ ስብሕት፡ በአርያም፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Saʾali lana Māryām māḫtota ʿālam sǝbǝḥt ba-ʾaryām</title>
```

Comment:
Is the title correct here? it is the same as LIT6287Mahtota.xml

Reply:
blatant mistake of mine!